calcom/cal.com
# feat: team availability - vertical selector and align times

## What does this PR do?
- In the teams availability view:
  - Alligns Timezones that are offset by minutes (i.e GMT +0:30)
  - Adds minutes to said timezones
  - Adds a verticle selector/guide to better view alligned timezones.  
- Fixes #17499 
- Fixes CAL-4680 

 Loom Video: https://www.loom.com/share/adc6fe94b0114ab2b5e2b5509118369c?sid=5b2dab2e-e4d0-4d0a-b723-3f32427a38f6

## Mandatory Tasks (DO NOT REMOVE)

- [ ] I confirm automated tests are in place that prove my fix is effective or that my feature works. (Doesn't seem to be any test suites to test the rest of the relevant code, which makes testing this quite difficult.) 

## How should this be tested?

Manually: 
1. Add timezones various teams in seed.ts
2. Run development mode via "yarn dx"
3. In the /availability?type=team, view the changes
   - Run the mouse along the calender days to move the verticle selector, it is hidden until the mouse stops hoving over the calender days.
   - Ensure the schedules that are offset by minutes are properly labelled and aligned. 
   
  /claim #17499
